### PR TITLE
Framework migration

### PR DESCRIPTION
**Pull request by SirM2z**
Migrate framework to ant.design and parcel

- [x] route code splitting
- home
   - [x] ui
   - [x] logic
- tutorial
  - [x] list
  - [x] Mnist
  - [x] AssetsClassification
- pipeline
  - [x] list
  - [x] detail
- job
  - [x] list
  - [x] detail
- plugin @yorkie 
  - [x] list
- setting @yorkie 
  - [x] list

### CODE REVIEW

**Comment by yorkie** [comment]
After having a discussion with @SirM2z @WenheLI, the migration excludes pipeline management.

File: src/pages/Job/index.jsx [hunk 1]
```diff
@@ -1,92 +1,122 @@
...
+        status: PipelineStatus[item.status]
+      }));
+    });
+  };
 
-  componentDidMount = async () => {
-    await this.fetchData(1);
-  }
+  useEffect(() => {
+    list(1).then(res => {
```
Comment (FeelyChau) [inline src/pages/Job/index.jsx:105]: I think we should ignore the promise result if DOM unmounted,  see https://juliangaramendy.dev/use-promise-subscription/

**Review by FeelyChau — CHANGES_REQUESTED** [review]

File: src/pages/Job/index.jsx (hunk 1 above)
Comment (SirM2z) [reply src/pages/Job/index.jsx:105]: It’s great

But it illustrates some of the react hook api issues

Comment (SirM2z) [reply src/pages/Job/index.jsx:105]: This will be resolved through a custom hook in the future pull requet

**Review by FeelyChau — APPROVED** [review]
LGTM.

**Review by yorkie — APPROVED** [review]

**Pull request merged**